assets/performanceprofile/tuned/openshift-node-performance

```diff
@@ -154,3 +159,11 @@ cmdline_idle_poll=+idle=poll
 cmdline_hugepages=+{{if .DefaultHugepagesSize}} default_hugepagesz={{.DefaultHugepagesSize}} {{end}} {{if .Hugepages}} {{.Hugepages}} {{end}}
 
 cmdline_additionalArg=+{{if .AdditionalArgs}} {{.AdditionalArgs}} {{end}}
+
+{{if .PerPodPowerManagement}}
```

Comment:
I am not sure how will this be translated to tuned final object in case both `PerPodPowerManagement` and `HighPowerConsumption` are set to true. 
Is that a valid scenario ?  and if so, how will the final `cmdline_pstate` line will look like in the generated tuned ?

Reply:
It is not possible to set both to true:

https://github.com/openshift/cluster-node-tuning-operator/blob/59698befa86989be3295f825c2faa3e8c5b5d4bb/pkg/performanceprofile/controller/performanceprofile/components/tuned/tuned.go#L176

Where I should document that?

Reply:
I think we should prevent that at profile creation time (aka validation hook)